test/Microsoft.AspNet.Mvc.FunctionalTests/TagHelpersTest.cs

```diff
@@ -54,38 +54,62 @@ public class TagHelpersTests
             Assert.Equal(expectedContent, responseContent);
         }
 
-        public static IEnumerable<object[]> TagHelpersAreInheritedFromViewStartPagesData
+        public static TheoryData TagHelpersAreInheritedFromGlobalImportPagesData
         {
             get
             {
-                var expected1 =
-@"<root>root-content</root>
-
-
-<nested>nested-content</nested>";
-                yield return new[] { "NestedViewStartTagHelper", expected1 };
-
-                var expected2 =
-@"layout:<root>root-content</root>
-
-
-<nested>nested-content</nested>";
-
-                yield return new[] { "ViewWithLayoutAndNestedTagHelper", expected2 };
-
-                var expected3 =
-@"layout:<root>root-content</root>
-
-
-page:<root/>
-<nested>nested-content</nested>";
-                yield return new[] { "ViewWithInheritedRemoveTagHelper", expected3 };
+                // action, expected
+                return new TheoryData<string, string>
+                {
+                    {
+                        "NestedGlobalImportTagHelper",
+                        string.Format(
+                            "<root>root-content</root>{0}{0}{0}<nested>nested-content</nested>",
+                            Environment.NewLine)
```

Comment:
We are specifically using multi-line strings since `Environment.NewLine` is unreliable for functional scenarios. The line endings are determined by your git settings and not by your OS. Need to go back to using multi-line strings.